app/controllers/assessments_controller.rb

```diff
@@ -70,8 +70,20 @@ def new
     @threshold_hash = jurisdiction.hierarchical_symptomatic_condition.threshold_condition_hash
     @translations = @assessment.translations
     @contact_info = jurisdiction.contact_info
-    @lang = permitted_params[:lang] if %w[en es es-PR so fr].include?(params[:lang])
-    @lang = 'en' if @lang.nil? # Default to english
+
+    # For backwards-compatibility reasons, we still want to support the old 2-letter language codes
+    legacy_language_mapping = {
```

Comment:
let's define this as a constant either at the top of the file or in a separate one

Reply:
good idea! done.

Reply:
not seeing any new commits, did you forget to push?

Reply:
Yes! Pushed now